pkgs/development/python-modules/networkx/default.nix

```diff
@@ -23,6 +24,10 @@ buildPythonPackage rec {
     pytest
   '';
 
+  passthru.tests = {
+    inherit sage;
```

Comment:
I don't agree with adding this. Sage is a large package with many dependencies. It's a huge step to go from networkx to sage.

Reply:
Upstream Sage is undergoing a modularization effort, so it is possible that next year we'll be able to add just Sage tests involving graphs here. In the meantime, though, I think Sage should only be added to "important" dependencies'  `passthru.tests` (e.g., Singular, GAP, Maxima, perhaps SymPy).

Reply:
It's OK if Sage breaks every once in a while, such is life for a project of this size. The only thing I would really like is to get a notification when a Hydra build fails in a branch, since breakage in `staging-next` is easy to miss.

Reply:
> It's OK if Sage breaks every once in a while, such is life for a project of this size. The only thing I would really like is to get a notification when a Hydra build fails in a branch, since breakage in `staging-next` is easy to miss.

Understandable, but that is a whole other issue to fix.